integration-tests/olminstall/suite/cluster_api_health.py

```
@@ -141,25 +154,80 @@ def openshift_console_route_unavailable_reason(*, probe: bool = True) -> str:
+def openshift_guest_rh_ai_route_tekton_unreachable_reason(*, probe: bool = True) -> str:
+    """When the cluster is an OCI guest, rh-ai.apps routes often fail DNS from Tekton (Playwright, MLflow)."""
+    if not probe or not _extended_eaas_infra_probes_enabled():
+        return ""
+    try:
+        result = oc_run(
+            ["whoami", "--show-console"],
+            check=False,
+            capture_output=True,
+            timeout=15,
+        )
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)